src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/Frame.cs

```diff
@@ -1422,6 +1430,15 @@ protected void EnsureHostHeaderExists()
             }
         }
 
+        private IPipe CreateRequestBodyPipe()
+            => ConnectionInformation.PipeFactory.Create(new PipeOptions
+            {
+                ReaderScheduler = ServiceContext.ThreadPool,
+                WriterScheduler = ConnectionInformation.InputWriterScheduler,
```

Comment:
Inline would be equivalent to the way we previously implemented the request and response body streams. That makes the most sense to me.